website/blog/2019-07-22-presto-unlimited-mpp-database-at-scale.md

```diff
@@ -0,0 +1,123 @@
+
+For more details, please see the design doc at [#12387](https://github.com/prestodb/presto/issues/12387)
+
+### Recoverable Grouped Execution
+
+Grouped execution also enables partial query failure recovery, as now each lifespan of the query is independent and can be retried independently. As illustrated in the following figure:
+
+![Remote Exchange](/img/blog/2019-07-22-presto-unlimited-mpp-database-at-scale/grouped_recovery.png)
+
+For more details, please see the design doc at [#12124](https://github.com/prestodb/presto/issues/12124)
+
+
+## Future Work
+
+We are also thinking about the following future work:
+- Fault-Tolerant Exchange Execution
+    * With exchange materialization and recoverable grouped execution, the only reason that a single worker failure can fail the query is during exchange stage. 
+    * In a long term, the solution is to implement MapReduce-style shuffle, or integrate with a fault-tolerant distributed shuffle service such as [Cosco](https://databricks.com/session/cosco-an-efficient-facebook-scale-shuffle-service) or [Crail](https://crail.incubator.apache.org/).
+    * In a short term, the plan is to support recoverability for each execution section. Thus the query can restart from the last checkpoint instead of the beginning.
+
+- Reliable and Scalable Coordinator
+    * Once query execution can survive all types of the worker failures, coordinator starts to become the single point of failure. 
+    * Coordinator high-availability and scalability is a crucial future work for Presto Unlimited project. 
+
+- Resource Management for Presto Unlimited
+    * With more larger queries onboard, we need to actively monitor whether the current resource management needs improvement or revamp for the potential workload shifts introduced by Presto Unlimited. 
+    * Besides providing retry unit at fine granularity, “lifespan” also provides smaller unit for resource management, and opens opportunities for fine-grained resource management. 
+
+
+## Parallel Databases Meet MapReduce
+
+Note with Presto Unlimited, Presto executes large ETL query in a similar way to MapReduce:
+
+![Presto Unlimited Execution Draft](/img/blog/2019-07-22-presto-unlimited-mpp-database-at-scale/presto_unlimited_james.png)
+
+In comparison, here is how query executed without Presto Unlimited:
+
+![Presto Nomral Execution Draft](/img/blog/2019-07-22-presto-unlimited-mpp-database-at-scale/presto_normal_james.png)
+
+During the last decade, the standard approach to support SQL at scale is to build parallel database on a MapReduce-like runtime: Tenzing [1], Hive [2], SCOPE [3], SparkSQL [4], F1 Query [5], etc.
+
+To the best of our knowledge, Presto Unlimited is the first attempt to meet parallel database and MapReduce in a different direction, as it brings MapReduce-style execution to the parallel database.
```

Comment:
Just to clarify, the main point of this sentence is that other engine built the engine with MR in mind, while Presto Unlimited is introducing MR into a mature engine?